cluster/swarm/cluster.go

```diff
@@ -242,11 +242,32 @@ func (c *Cluster) Image(IDOrName string) *cluster.Image {
 	return nil
 }
 
-// RemoveImage removes an image from the cluster
-func (c *Cluster) RemoveImage(image *cluster.Image) ([]*dockerclient.ImageDelete, error) {
+// RemoveImages removes all the images that match `name` from the cluster
+func (c *Cluster) RemoveImages(name string) ([]*dockerclient.ImageDelete, error) {
 	c.Lock()
 	defer c.Unlock()
-	return image.Engine.RemoveImage(image)
+
+	out := []*dockerclient.ImageDelete{}
+	errs := []string{}
+	var err error
+	for _, n := range c.engines {
+		for _, image := range n.Images() {
+			if image.Match(name) {
+				content, err := image.Engine.RemoveImage(image, name)
+				if err != nil {
+					errs = append(errs, fmt.Sprintf("%s: %s", image.Engine.Name, err.Error()))
+					continue
+				}
+				out = append(out, content...)
+			}
```

Comment:
This case:
Node1: testimage:ok
Node2:testimage:latest
`docker_swarm rmi testimage`
 I think the ouput will contain "No such image: testimage:latest" respond by Node1.